### ISSUE TITLE
Can't make <ol> to render

### ISSUE THREAD

**Issue by KulovacNedim** [yhatt/jsx-slack#258]
Hi @yhatt - I'm facing an issue with the `<ol>` element, I wonder if you could help me? It looks to me like a bug related to `jsx-slack`. 

Here is what I'm trying to do:
```
await client.chat.postMessage({
      token: botToken,
      channel,
      text: 'Sorry about that. Please report an issue to your administrator.',
      blocks: JSXSlack(
        <Blocks>
          <Section>
            <ol type="1">
              <li>First item</li>
              <li>Second item</li>
            </ol>
          </Section>
        </Blocks>
      )
    });
```
and I'm getting the following error:
```
[ERROR]  bolt-app ERROR: TypeError: Cannot convert object to primitive value
```

Going further, if I remove the second item, and leave just the first one, I'm getting following error:
```
[ERROR]  bolt-app ERROR: TypeError: u[t].toString is not a function
```

If I rename `<ol>` to `<ul>` (and remove `type` of course), I'm getting properly displayed unordered list in Slack. So the problem seems to be only with an ordered list. The other elements and blocks are rendered fine. 
Note: We recently switched from `@speee-js/jsx-slack` to `jsx-slack` - not sure if this is when an issue with `ol` started to happen, but I think it could be the case.

The project is written in typescript, here is `tsconfig.json`: 
```
{
  "compilerOptions": {
    "target": "ESNext",
    "module": "commonjs",
    "lib": ["dom", "es6"],
    "allowJs": true,
    "jsx": "react-jsx",
    "jsxImportSource": "jsx-slack",
    "isolatedModules": false,
    "strict": true,
    "noImplicitAny": true,
    "moduleResolution": "node",
    "baseUrl": "src",
    "outDir": "dist",
    "esModuleInterop": true,
    "resolveJsonModule": true,
    "sourceMap": true,
    "paths": {
      "@/*": ["./*"]
    }
  },
  "include": ["src"],
  "exclude": ["dist", "node_modules"]
}
```

and `package.json`:
```
"dependencies": {
    "@sendgrid/mail": "^7.5.0",
    "@slack/bolt": "^3.8.1",
    "@types/mocha": "^8.2.0",
    "@types/node": "^14.17.27",
    "@types/puppeteer": "^5.4.4",
    "axios": "^0.24.0",
    "date-fns": "^2.25.0",
    "dotenv": "^8.2.0",
    "jsx-slack": "^4.5.2",
    "mixpanel": "^0.14.0",
    "module-alias": "^2.2.2",
    "puppeteer": "^10.4.0",
    "typescript": "4.5.2"
  },
  "devDependencies": {
    "@commitlint/cli": "^8.3.5",
    "@commitlint/config-conventional": "^8.3.4",
    "@typescript-eslint/eslint-plugin": "^3.2.0",
    "@typescript-eslint/parser": "^3.2.0",
    "assert": "^2.0.0",
    "eslint": "^7.2.0",
    "eslint-config-prettier": "^6.11.0",
    "eslint-plugin-prettier": "^3.1.3",
    "husky": "^4.2.5",
    "lint-staged": "^10.2.9",
    "mocha": "^8.3.0",
    "prettier": "^2.0.5",
    "ts-node": "^8.10.2",
    "ts-node-dev": "^1.0.0-pre.44",
    "ts-node-paths": "^1.0.1"
  },
  "engines": {
    "node": "14.x"
  }
```

Thanks!

**Comment by yhatt** [comment]
Thank you for your report 😄

I found out a tricky edge case that came from difference of behavior between JSX runtime for `react-jsx` mode and JSX conversion for `react` mode. 

I'll work to fix this, but if you are in a hurry, you can apply a workaround by switching to `react` mode temporarily.

```jsonc
{
  "compilerOptions": {
    "jsx": "react",
    "jsxFactory": "JSXSlack.h",
    "jsxFragmentFactory": "JSXSlack.Fragment",
    // ...
  }
}
```

<details>


When user had JSX like this:

```html
<ul type="1">
  <li>a</li>
  <li>b</li>
</ul>
```

In `react` mode, the above JSX will be transformed like this:

```javascript
JSXSlack.h('ul', { type: '1' },
  JSXSlack.h('li', null, 'a'),
  JSXSlack.h('li', null, 'b'),
)
```

And in `react-jsx` mode:

```javascript
import { jsx } from 'jsx-slack/jsx-runtime'

jsx(
  'ul',
  {
    type: '1',
    children: [
      jsx('li', { children: 'a' }, undefined),
      jsx('li', { children: 'b' }, undefined),
    ]
  },
  undefined
)
```

There are clear differences about how to pass children elements between `react` and `react-jsx` mode.

jsx-slack will try to convert JSX to HTML string for internal like `'<ul type="1"></ul>'`. The second argument in `react` mode has attributes only user specified (`type`), but `react-jsx` mode is including children as an `children` attribute.

A reported error has caused by trying jsx-slack to generate a string expression of `children` attribute forcibly, like `` `<ul type="1" children="${[<li>a</li>, <li>b</li>].toString()}">` ``. HTML tags such as `<li>` will return an object with an empty prototype (`Object.create(null)`) so it cannot respond to `Object.toString()`.

</details>

**Comment by yhatt** [comment]
I've patched and shipped the latest release jsx-slack v4.5.3. Could you try to update your jsx-slack dependency, and confirm rendering of list?

**Comment by KulovacNedim** [comment]
Hey @yhatt - thanks a ton! 

It works fine in v4.5.3
